### ISSUE TITLE
Unsupported granule type - need improved file handling and reporting

### ISSUE THREAD

**Issue by michaelleerilee** [SpatioTemporal/STAREPandas#91]
The following is a little obscure

![image](https://user-images.githubusercontent.com/6934724/166693416-2f531550-4c22-4ace-94cb-70450f30f346.png)

**Comment by NiklasPhabian** [comment]
Yes. I guess we should add a bit more info in the UnsupportedFileError.
What this means though is that we have not added support for this product

**Comment by NiklasPhabian** [comment]
https://github.com/SpatioTemporal/STAREPandas/blob/master/starepandas/io/granules/__init__.py#L10
